src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_general_operations.py

```diff
@@ -197,15 +197,17 @@ class MonitorCloneStorageAccountAcrossSubsScenarios(ScenarioTest):
     @unittest.skip('Accross subs are not supported now.')
     # @live_only()
     @ResourceGroupPreparer(name_prefix='cli_test_metric_alert_clone')
-    def test_monitor_clone_storage_metric_alerts_across_subs_scenario(self, resource_group):
+    @ResourceGroupPreparer(name_prefix='cli_test_metric_alert_clone',
+                           parameter_name='another_resource_group', subscription='1c638cf4-608f-4ee6-b680-c329e824c3a8')
+    def test_monitor_clone_storage_metric_alerts_across_subs_scenario(self, resource_group, another_resource_group):
         self.kwargs.update({
             'alert': 'alert1',
             'sa': self.create_random_name('sa', 24),
             'sa2': self.create_random_name('sa', 24),
             'ag1': 'ag1',
             'rg': resource_group,
             'ext_sub': '1c638cf4-608f-4ee6-b680-c329e824c3a8',
-            'ext_rg': self.create_random_name('test_rg', 24),
+            'ext_rg': another_resource_group,
         })
 
         self.cmd('group create -l eastus -n {ext_rg} --subscription {ext_sub}')
```

Comment:
Unnecessary to create rg again as you create it in `ResourceGroupPreparer`

Reply:
OK~